test/run-tests

```diff
@@ -180,16 +180,13 @@ class Task:
                 with open(setup_file, 'w') as f:
                     json.dump([play], f)
 
-                playbooks.append(setup_file)
-
             # Fedora's standard test invocation spec mandates a
```

Comment:
this needs to be changed to

    mandates running all playbooks matching

to match to the change from running just one playbook to several.

Reply:
Fixed. Thanks.